src/python/pants/backend/python/goals/export.py

```diff
@@ -401,6 +564,20 @@ async def export_virtualenv_for_resolve(
             editable_local_dists_digest=editable_local_dists_digest,
         ),
     )
+
+    # Add generated Python sources from codegen targets to the virtualenv.
+    if export_subsys.options.py_generated_sources:
+        if export_subsys.options.py_resolve_format == PythonResolveExportFormat.mutable_virtualenv:
+            export_result = await add_codegen_to_export_result(
+                request.resolve, export_result, codegen_setup
+            )
+        else:
+            logger.info(
```

Comment:
I guess if we switch the default to True we'll want to get rid of this logging altogether? Maybe we should just kill it now. It seems weird to need to say "that thing that is documented to not affect the current case is indeed not affecting it"?

Reply:
Yes we would want to get rid of it. I was concerned with users having `--export-py-generated-sources` but exporting to the symlink'ed virtualenv and then wondering why there were no generated sources.

But we could just rely on them reading the docs at that point instead of always spamming the console with this info message.

Reply:
I removed the message. If users complain enough, can always add it back.